bin/cost_changes.sh

```diff
@@ -69,3 +68,10 @@ echo $totals_file;
 echo $diff_file;
 echo $diffp_file;
 echo $hilites_file;
+
+# Special handling until the budget is set in 2025:
+prev_budget_report=`echo "$all_reports" | grep -Po '\S+costreport_20240801.tsv'`
```

Comment:
These lines generate the extra data and puts them in 2 separate files that can be pasted into the google sheet version of the report.